topic-offset-service/README.md

```diff
@@ -0,0 +1,91 @@
+<!---
+Copyright 2018-2021 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--->
+
+# <img src="../logos/logo.svg" width="180">
+
+# Topic Offset Service
+
```

Comment:
Opening sentence along the lines of "Watches kafka topis and reports offsets for messages that satisfy some prerequisite"